Dragonflight/PaladinHoly.lua

```diff
@@ -1528,6 +1535,33 @@ spec:RegisterAbilities( {
     },
 
 
+    stoneform = {
```

Comment:
This already exists in Classes.lua and doesn't need to be duplicated.  I see you're putting it on Defensives, but that can be done in Classes.lua as well.